src/main/java/lrbg/codriver/data/Event.java

```diff
@@ -0,0 +1,39 @@
+package lrbg.codriver.data;
+
+import java.time.LocalDate;
+
+/**
+ * Represents an event task.
+ */
+public class Event extends Task {
+    /** The start date of the event. */
+    private final LocalDate from;
+    /** The end date of the event. */
+    private final LocalDate to;
```

Comment:
Perhaps you may consider renaming the variables `from` and `to` to `startDate` and `endDate` respectively to make it more intuitive for the user?